Library/Homebrew/cmd/install.rb

```diff
@@ -338,3 +339,23 @@ def install_formula(f)
     ofail e.message
   end
 end
+
+def forbidden_license_check(f)
+  forbidden_licenses = ENV["HOMEBREW_FORBIDDEN_LICENSES"].split(" ")
```

Comment:
```suggestion
  forbidden_licenses = Homebrew::EnvConfig.forbidden_licenses.split(" ")
```

or could even add a custom method there to make it return an Array?